packages/cli/README.md

```diff
@@ -259,10 +259,9 @@ DESCRIPTION
   and access scopes.
   - Building and serving your app and app extensions.
 
-  If you're using the PHP or Ruby app template, then you need to complete the following steps before you can preview
+  If you're using the Ruby app template, then you need to complete the following steps before you can preview
```

Comment:
QQ: Is this readme generated from somewhere? Seems to be from `dev.ts`

Reply:
It is. I think CI checks will nudge you to do this but `pnpm refresh-readme` should do the job.